UnityProject/Assets/Scripts/Messages/Client/Admin/RequestRespawnPlayer.cs

```diff
@@ -37,8 +37,12 @@ void VerifyAdminStatus()
 
 	void TryRespawn(ConnectedPlayer deadPlayer)
 	{
-		UIManager.Instance.adminChatWindows.adminToAdminChat.ServerAddChatRecord(
-			$"{PlayerList.Instance.GetByUserID(Userid).Name} respawned dead player {deadPlayer.Name}", Userid);
+		string msg = $"{PlayerList.Instance.GetByUserID(Userid).Name} respawned dead player {deadPlayer.Name}";
```

Comment:
this logic is duplicated in quite a few places, refactor it to a method

Reply:
Yeah ive moved it to its own method.